### PR TITLE
Add mozillavpn workers and update appservices

### PR DESCRIPTION
**Pull request by MasterWayZ**

### CODE REVIEW

File: modules/dirs/manifests/builds.pp [hunk 1]
```diff
@@ -5,9 +5,6 @@
 class dirs::builds {
 
     $builds_dir = $facts['os']['macosx']['version']['major'] ? {
-        '13.5' => '/opt/builds',
```
Comment (MasterWayZ) [inline modules/dirs/manifests/builds.pp:8]: These are redundant, as the default is set to `/opt/builds`

File: modules/dirs/manifests/tools.pp [hunk 2]
```diff
@@ -5,9 +5,6 @@
 class dirs::tools {
 
     $tools_dir = $facts['os']['macosx']['version']['major'] ? {
-        '13.5' => '/opt/tools',
```
Comment (MasterWayZ) [inline modules/dirs/manifests/tools.pp:8]: These are redundant, as the default is set to `/opt/tools`

File: modules/talos/manifests/init.pp [hunk 3]
```diff
@@ -8,10 +8,8 @@
     case $facts['os']['name'] {
         'Darwin': {
             $builds_dir = $facts['os']['macosx']['version']['major'] ? {
-                '13.5' => '/opt/builds',
```
Comment (MasterWayZ) [inline modules/talos/manifests/init.pp:11]: These are redundant, as the default is set to `/opt/builds`

**Comment by MasterWayZ** [comment]
Note: moving appservices to ::worker is intentional

**Review by aerickson — APPROVED** [review]
LGTM. Very nice.

**Pull request merged**